apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx

```diff
@@ -169,28 +169,29 @@ export const SignInMethodLanding: FunctionComponent<SignInMethodLandingPropsInte
                                     } }
                                 />
                             ) }
-                            { !hiddenOptions.includes(LoginFlowTypes.SECOND_FACTOR_SMS_OTP) && (
```

Comment:
Can't we put `SECOND_FACTOR_SMS_OTP` in the hiddenOptions?

Reply:
+1. Let's add this to the `hiddenOptions` array.

Reply:
Let's use the deployment toml config `hiddenAuthenticators` array to hide this. We can send a follow-up PR.
Ref: https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx#L206

Then you'll be able to remove the additional filter logic.